documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -131,7 +131,10 @@ on GitHub.
   only delegate to `TestDescriptor.Type` constants now.
 * Introduced `TestDescriptor.prune()` and `TestDescriptor.pruneTree()` which allow engine authors
   to customize what happens when pruning is triggered by the JUnit Platform.
-
+* `TestIdentifier` now also uses the new enumeration `TestDescriptor.Type` to store the
```

Comment:
How did this end up in your commits?

Please rebase your work on `master` and make sure that only your changes are included in your PR.